seatunnel-apis/seatunnel-api-spark/src/main/scala/org/apache/seatunnel/spark/batch/SparkBatchExecution.scala

```diff
@@ -80,9 +74,7 @@ object SparkBatchExecution {
     ds.createOrReplaceTempView(tableName)
   }
 
-  private[seatunnel] def registerInputTempView(
```

Comment:
Just a nit: I think we can keep it as it is, when the function has many args, the first form is better.
I am not sure if others prefer to the second form.
```
private[seatunnel] def registerInputTempView(
      source: BaseSparkSource[Dataset[Row]],
      environment: SparkEnvironment): Unit = {
```
```
private[seatunnel] def registerInputTempView(source: BaseSparkSource[Dataset[Row]], environment: SparkEnvironment): Unit = {
```

Reply:
Do I want to restore it? I want to hear your suggestions.

Reply:
I also feel that the first one will be better.